localizer.ipynb

```diff
@@ -21,50 +21,81 @@
     "\n",
     "import tello\n",
     "import detection\n",
+    "import cv2\n",
```

Comment:
Notebook's broken. Try to open and it says "Unable to open 'localizer.ipynb' Unexpected string in JSON at position 705"